end-to-end.md

```diff
@@ -0,0 +1,191 @@
+
+## Download source data files
+
+    # test S3 access
+    aws --profile development s3 ls s3://find-insights-input-data-files
+
+    cd dataingest/addtodb
+    ./download-data-2i.sh
+
+    cd ../geoname
+    ./download.sh
+
+    cd ../spatial
+    ./download.sh
+
+    cd ../postcode
+    ./download.sh
+
+    cd ../..
+
+## Run the ingest
+
+The ingest scripts are meant to talk to a local postgres instance, so set the environment variables as below.
+
+`PGPASSWORD` is the password for the `insights` user, and `POSTGRES_PASSWORD` is the postgres superuser password
+(our convention copied from the Postgres docker image).
+
+    export PGHOST=localhost
+    export PGPORT=5432
+    export PGDATABASE=census
+    export PGUSER=insights
+    export PGPASSWORD=apassword
+
+    PGPASSWORD=$POSTGRES_PASSWORD psql --dbname postgres --username postgres -c "CREATE USER $PGUSER WITH PASSWORD '$PGPASSWORD' CREATEDB"
+
+    ./indigestion.sh
+
+## Export Database
+
+This export step also talks to the local postgres instance, so keep the environment variables from the Ingest step above.
+
+    cd dataingest/dbsetup
+    today=$(date +%Y%m%d)
+    pg_dump --exclude-table=spatial_ref_sys > census-${today}.sql
+
+## Provision RDS
+
+This step doesn't need to be done if there is already an RDS instance in the `develop` environment.
+
+    cd <dp-setup-repo>
+    cd terraform/dp-geodata-api-postgres
+    gpg -d < develop.tfvars.asc > develop.tfvars
+
+    terraform init
+    terraform workspace select develop
+
+    terraform plan -var-file=develop.tfvars
+
+## Set up SSH tunnel
+
+You need the RDS instance hostname and port when you set up the tunnel.
+You can grab this information like this:
+
+    cd <dp-setup-repo>
+    cd terraform/dp-geodata-api-postgres 
+    terraform output
+
+Once you have the RDS instance endpoint details, and run this in a separate terminal on your workstation:
+
+    dp ssh develop web 1 --port 5555:<endpoint-host>:<endpoint-port>
+
+Now back to your working terminal, set the environment to point to the tunnel and to use login details for the RDS instance.
+The superuser password is found in `develop.tfvars`.
+
+        export PGHOST=localhost
+        export PGPORT=5555
+        export PGDATABASE=census
+        export PGUSER=insights
+        export PGPASSWORD=<insights-password>
+        export POSTGRES_PASSWORD=<superuser-password>
+
+        # test connectivity to db; this should list 4 databases
+        $ PGPASSWORD=$POSTGRES_PASSWORD psql --dbname postgres --username postgres -c '\l'
+
+## Import into RDS
+
+Verify your environment variables point to the tunnel and use the RDS credentials, then:
+
+    cd <dp-find-insights-poc-api repo>
+    cd dataingest/dbsetup
+    
+    # this is the sql dump file created in the Export Database step above
+    ./creatdb.sh census-20220422.sql
+
+## Start local API
+
+In another terminal on your workstation, set the environment to point to the tunnel and
+use the RDS credentials.
```

Comment:
maybe worth a reference to the env vars on lines 132-140 in the 'set the environment to the tunnel' bit? I know its like ten lines above it but I read it and had to think for a few seconds